api/client.go

```diff
@@ -44,6 +44,43 @@ func NewClient(c *ClientConfig, httpClient httpClient) *Client {
 	}
 }
 
+func (c *Client) Port() int {
```

Comment:
Added bonus cleanup here. Nice.

Reply:
🎉 thanks john! It was a fun work-around to think through. The testutil function `StartCTS` runs using a config file written to disk by the caller, but to create the client within the helper it requires the port.

Reply:
it worked out nicely to do this refactor after @lornasong had built out the client for easy usage 💯